services/email/handlers/processEmails.js

```diff
@@ -0,0 +1,60 @@
+import AWS from "aws-sdk";
+import { dynamoConfig } from "cmscommonlib";
+
+const dynamoDb = new AWS.DynamoDB.DocumentClient(dynamoConfig);
+
+const oneMacTableName = process.env.IS_OFFLINE
+  ? process.env.localTableName
+  : process.env.oneMacTableName;
+
+export const main = async (event, context, callback) => {
+  console.log("Received event (stringified):", JSON.stringify(event, null, 4));
+
+  // get emails in certain statuses
+  const queryParams = {
+    TableName: oneMacTableName,
+    IndexName: "GSI2",
+    KeyConditionExpression: "GSI2pk = :pk",
+    ExpressionAttributeValues: {
+      ":pk": "Send",
+    },
+  };
+  try {
+    const result = await dynamoDb.query(queryParams).promise();
+    console.log("results are: ", result);
```

Comment:
Is this just querying but not doing anything? I guess im not following the entire workflow

Reply:
So... at this time, yes, processEmails queries the database to show how to get the emails for a given status, BUT, I have no idea which statuses they want to do anything with and what to do with them.

The story is to "retry failed messages," BUT, the soft failures that we would conceivably want to retry (as in, it's not a bad email address and will always be bad) have actually already been retried by AWS SES several times before the final reporting.  So.... story AC is already met, but I thought it might be good to leave in the "how" to discover a truly failed email.